src/components/graphics/GraphicComponent.vue

```diff
@@ -119,7 +119,7 @@ export default {
 		},
 
 		nativeMosaicAliasName() {
-			return http.networkCurrecy.mosaicId.namespace;
+			return http.networkCurrecy.namespace[2];
```

Comment:
What is `namespace[0]` and `namespace[1]`?

Reply:
https://github.com/nemgrouplimited/symbol-explorer/blob/a39aab9096e5ff33e6cfdb4dfcea28ddc80205da/src/infrastructure/http.js#L52

namespace[0] = `symbol`
namespace[1] = `xym`
namespace[2] = `symbol.xym`

maybe we should separate it to ?

```
namespace[0] = `symbol`
namespace[1] = `xym`
namespaceFull = `symbol.xym`
```